### PR TITLE
Add conda-deps package

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
@conda-forge/help-python please review

**Comment by sebastian-luna-valero** [comment]
@conda-forge/staged-recipes please review

File: recipes/conda-deps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+test:
+  commands:
+    - conda_deps --help
+
+about:
+  home: https://github.com/cgat-developers/conda-deps
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Generate conda environment files from Python source code'
```
Comment (ocefpaf) [inline recipes/conda-deps/meta.yaml]: Nice! There may be some overlap with https://github.com/pyoceans/nbrr, let's talk and get the best of the two into one package.

**Comment by sebastian-luna-valero** [comment]
Many thanks @ocefpaf 

I will have a look at nbrr next week.

**Pull request merged**